### DISCUSSION TITLE
how to registerSetup<MvxSetup> without MvxFormsAppCompatActivity<MvxSetup>

### DISCUSSION THREAD

**Discussion by JaehwanMango** [MvvmCross/MvvmCross#4671] [Q&A] [answered]
### How to register `MvxSetup`?
I trying to migration to MAUI
I can't use  `MainActivity : MvxFormsAppCompatActivity<MvxSetup>` in maui

I lost the only way i know `register MvxSetup`
How can i register `MvxSetup` in Maui?

**Comment by Cheesebaron** [comment, marked as answer]
We don't support MAUI the UI framework currently. No plans to do so in the short term either.
